src/perl/vdotest/VDOTest/Dmsetup.pm

```diff
@@ -64,6 +65,35 @@ sub testBasicOps {
   assertNe("", $EVAL_ERROR, "unknown message failed to generate an error");
 }
 
+###############################################################################
+# Test dmsetup message for displaying config information
+#
+##
+sub testConfig {
+  my ($self) = assertNumArgs(1, @_);
+  my $device = $self->getDevice();
+  my $path = $device->getVDODevicePath();
+
+  my $output = $device->runOnHost("sudo dmsetup message $path 0 config");
+  $output = "config : " . $output;
+  my $yaml = yamlStringToHash($output);
+
+  my $slabBits = $self->{slabBits};
+  if (!defined($slabBits)) {
+    # default slab bits from vdoformat
+    $slabBits = 19;
+  }
+
+  my $config = $yaml->{"config"};
+
+  assertEq($config->{"physicalSize"}, int($device->{storageDevice}->getSize()));
+  assertEq($config->{"logicalSize"}, $device->{logicalSize});
+  assertEq($config->{"slabSize"}, 1 << $slabBits);
+  assertEq($config->{"index"}->{"memorySize"}, $self->{memorySize});
+  assertEq($config->{"index"}->{"isSparse"}, $device->{sparse});
+
+}
```

Comment:
You don't need a blank line at the end of a block.

Reply:
Will remove